protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -285,11 +286,11 @@ definitions:
       - type
     properties:
       type:
-        description: "the event type"
+        description: "The event type - should be a static string. Keep in mind that all strings are shared across all connectors."
         type: string
       value:
         type: string
-        description: "the value of the event"
+        description: "The value of the event - can be an arbitrary string. In case the value is numeric, it should be converted to a string. Casting for analytics purposes can happen in the warehouse."
```

Comment:
Shouldn't these same changes be applied to the V0 file as well?

Reply:
They absolutely should be, adjusted. Thanks for noticing!